src/Courses/CourseDetailsPage.tsx

```diff
@@ -73,7 +75,7 @@ export const CourseDetailsPage: React.FC<CourseDetailsPageProps> = () => {
                                     aria-label="Enrollment link"
                                     aria-describedby="basic-addon2"
                                     ref={textAreaRef}
-                                    value={`https://${enrollUrl}`}
+                                    value={`http://${enrollUrl}`}
```

Comment:
Yeah I noticed this last night, it would be nice if we could get the protocol from the browser